### PR TITLE
[Wallet] Implement most of verification loading screen and skip modal

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

- Add Lottie package and new loading animation 
- Implement skip modal in verification education screen
- Implement most of verification loading screen
- Remove unused dropdown modal package

### Tested

Relevant screens tested locally
still TODO test iOS, will do after done verification implementation

### Related issues

- For #1523 

### Backwards compatibility

Yes

wa-verloading
wa-ver-skip-modal

### CODE REVIEW

File: packages/mobile/ios/Podfile.lock [hunk 1]
```diff
@@ -389,8 +392,6 @@ PODS:
     - React
   - RNPermissions (2.0.2):
     - React
-  - RNRandomBytes (3.0.0):
```
Comment (cmcewen) [inline packages/mobile/ios/Podfile.lock]: Do you know why this is being removed?

File: packages/mobile/package.json [hunk 2]
```diff
@@ -99,8 +101,7 @@
     "react-native-keyboard-aware-scroll-view": "^0.9.1",
     "react-native-localize": "^1.3.0",
     "react-native-mail": "^4.0.0",
-    "react-native-modal": "^11.4.0",
-    "react-native-modal-dropdown": "^0.7.0",
```
Comment (cmcewen) [inline packages/mobile/package.json:103]: 👍

File: packages/mobile/src/components/Carousel.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ interface OwnProps {
 
 export interface CarouselItem {
   text: string
-  icon?: React.ComponentType
+  icon?: any
```
Comment (cmcewen) [inline packages/mobile/src/components/Carousel.tsx]: 😢

File: packages/mobile/src/verify/VerificationEducationScreen.test.tsx [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
-    const tree = renderer.create(
+    const { getByTestId, toJSON } = render(
       <Provider store={store}>
         <VerificationEducationScreen />
       </Provider>
     )
-    expect(tree).toMatchSnapshot()
+    expect(toJSON()).toMatchSnapshot()
+
+    // And snapshot again after showing the modal
```
Comment (cmcewen) [inline packages/mobile/src/verify/VerificationEducationScreen.test.tsx:19]: 🙏

File: packages/mobile/src/verify/VerificationLoadingScreen.tsx [hunk 5]
```diff
@@ -1,34 +1,110 @@
...
+    verificationFailed: state.identity.verificationFailed,
+    account: currentAccountSelector(state),
+    underlyingError: errorSelector(state),
+  }
+}
+
+class VerificationLoadingScreen extends React.Component<Props> {
+  static navigationOptions = { header: null }
+
+  onCancel = () => {
```
Comment (cmcewen) [inline packages/mobile/src/verify/VerificationLoadingScreen.tsx:71]: what should happen here?

File: packages/mobile/src/components/Carousel.tsx (hunk 3 above)
Comment (jmrossy) [reply packages/mobile/src/components/Carousel.tsx]: Will fix :)

File: packages/mobile/ios/Podfile.lock (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/ios/Podfile.lock]: No but we don't seem to use it. Not sure why my cocopods removed it when Jean's didn't

File: packages/mobile/src/verify/VerificationLoadingScreen.tsx (hunk 5 above)
Comment (jmrossy) [reply packages/mobile/src/verify/VerificationLoadingScreen.tsx:71]: We cancel the verification saga and go back to the education screen. Will do in next PR

File: packages/mobile/ios/Podfile.lock [hunk 6]
```diff
@@ -708,4 +713,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 0d3a8018c6a563abba8559d74620bf8b8b370fe8
 
-COCOAPODS: 1.7.5
+COCOAPODS: 1.8.4
```
Comment (jeanregisser) [inline packages/mobile/ios/Podfile.lock]: Looks like `pod install` was run instead of `bundle exec pod install`.

**Review by jeanregisser — DISMISSED** [review]
Great 👍 

Just need to address the `Podfile.lock` issue.

File: packages/mobile/ios/Podfile.lock (hunk 6 above)
Comment (jmrossy) [reply packages/mobile/ios/Podfile.lock]: It was yes! what's the difference? Is it that bundle exec runs the pod version specified in this lock?

Comment (jeanregisser) [reply packages/mobile/ios/Podfile.lock]: There's a fixed version of CocoaPods in `Gemfile.lock` so all devs are using the same one.
When you run `pod install` it runs your system installed CocoaPods, and when you run `bundle exec pod install` it runs the version specified in `Gemfile.lock` (which is recommended).

Comment (jmrossy) [reply packages/mobile/ios/Podfile.lock]: Fixed

**Review by jeanregisser — APPROVED** [review]
All good! Ship it 👍

**Pull request merged**